### DISCUSSION TITLE
Handle products from multiple clients

### DISCUSSION THREAD

**Discussion by andreaskviby** [unopim/unopim#16] [Ideas]
We collect products from 34 different clients and need to separate them, possible?

**Comment by deepakk-webkul** [comment]
@andreaskviby Yes, it's possible to separate products from different clients in UnoPim. You can organize products by client using categories and channel filters for importing and exporting product data. This feature is already included in our roadmap and will be implemented in an upcoming release.
Suggestion:
The preferred method would be to create separate channels for each client. This allows you to keep each client's product content distinct and well-organized within their respective channels.

**Comment by andreaskviby** [comment]
Ok so to make it clear. Let’s say all our clients insert their items in
the PIM and they must be able to edit and manage their own products in
their channel. They I want them to be able to send the products to a
channel which will send it to let’s say Woocommerce or other APIs

Mvh / regards 

Andreas Kviby
***@***.***


On September 13, 2024, Tin ***@***.***> wrote:
> Yes, it's possible to separate products from different clients in
> UnoPim. You can organize products by client using categories and
> channel filters for importing and exporting product data. This feature
> is already included in our roadmap and will be implemented in an
> upcoming release.
>  Suggestion:
>  The preferred method would be to create separate channels for each
> client. This allows you to keep each client's product content distinct
> and well-organized within their respective channels.
>

—
Reply to this email directly, view it on GitHub
<https://github.com/orgs/unopim/discussions/16#discussioncomment-
10635937>, or unsubscribe <https://github.com/notifications/unsubscribe-
auth/AAMWXVDFAATYY4YKGRXKGA3ZWKYB5AVCNFSM6AAAAABOEEVSKOVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANRTGU4TGNY>.
You are receiving this because you authored the thread.Message ID:
***@***.***>

**Comment by deepakk-webkul** [comment]
@andreaskviby yes, all your clients can insert, edit, and manage their own products within their dedicated channels in the PIM. Each client will have their own channel where they can handle their product data in the product grid.

FYI, The products will be visible to all the clients they will have to update the product value in their particular channel. It's a centralized PIM. For now , we do not provide any option for managing only single channel data for a particular user.

Once the products are ready, they can send them to the appropriate external platforms, such as WooCommerce or other APIs, using the export functionality tied to their channel. This will allow seamless integration and control for each client.

Feel free to let me know if you'd like more details or if there’s anything else you need!

**Comment by andreaskviby** [comment]
Ok, can I write to someone privately to make clear that we could more
the products backend to this? We have thousands and thousands of
products from hundreds or customers in 11 countries soon that we need to
manage. Maybe it would be a great show case for you?

Mvh / regards 

Andreas Kviby
***@***.***


On September 13, 2024, Tin ***@***.***> wrote:
> Yes, it's possible to separate products from different clients in
> UnoPim. You can organize products by client using categories and
> channel filters for importing and exporting product data. This feature
> is already included in our roadmap and will be implemented in an
> upcoming release.
>  Suggestion:
>  The preferred method would be to create separate channels for each
> client. This allows you to keep each client's product content distinct
> and well-organized within their respective channels.
>

—
Reply to this email directly, view it on GitHub
<https://github.com/orgs/unopim/discussions/16#discussioncomment-
10635937>, or unsubscribe <https://github.com/notifications/unsubscribe-
auth/AAMWXVDFAATYY4YKGRXKGA3ZWKYB5AVCNFSM6AAAAABOEEVSKOVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANRTGU4TGNY>.
You are receiving this because you authored the thread.Message ID:
***@***.***>

**Comment by saurav-webkul** [comment]
Hi @andreaskviby 

Feel free to send us a ticket here https://webkul.uvdesk.com/en/customer/create-ticket/ and we will contact you. In type, you can choose Pre Sale query. In the ticket, you can also share your WhatsApp number as well so we can connect directly. 

I will be waiting for your ticket. 

Thanks
